css/css-sizing/aspect-ratio/block-aspect-ratio-038.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<title>CSS aspect-ratio: The definite max-width should win the automatic content-based minimum width</title>
+<link rel="author" title="Cathie Chen" href="mailto:cchen@igalia.com">
+<link rel="help" href="https://drafts.csswg.org/css-sizing-4/#aspect-ratio-minimum">
+<link rel="match" href="../../reference/ref-filled-green-100px-square.xht" />
+<meta name="assert" content="The definite max-width should win the automatic content-based minimum width.">
+
+<p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
+
+<div style="background: green; height: 100px; aspect-ratio: 2 / 1; max-width: 100px;">
```

Comment:
`height:100px` transferred through `aspect-ratio: 2 / 1` says to give it a 200px width, the auto min size says to give it a 200px width, but max-width should beat both of those. LGTM.